content/en/community/hackathons/sessions/baby-steps/sol/02-rot13/kraft.yaml

```diff
@@ -0,0 +1,20 @@
+---
+specification: '0.5'
+name: 01-echo-back
+unikraft:
+  version: 0.10.0
+targets:
+  - architecture: x86_64
+    platform: kvm
+libraries:
+  lwip:
+    version: 0.10.0
```

Comment:
I think if we need a lwip version, that should be added to the rest of the exercises too

Reply:
lwip can be safely set to `stable` as well